src/ShapeCrawler/Shapes/IShape.cs

```diff
@@ -30,6 +30,11 @@ public interface IShape
     /// </summary>
     int Height { get; set; }
 
+    /// <summary>
+    /// Gets the rotation of the shape in degrees.
+    /// </summary>
+    double RotationDegrees { get; }
```

Comment:
What about simply calling it `Rotation`?

The idea is when it is possible to name properties as they are in PowerPoint application:

![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/5969304/0604f512-3853-474b-b1dc-98e5b296bbbd)